target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-d7800.dts

```diff
@@ -80,11 +94,20 @@
 				};
 			};
 
-			mdio0_pins: mdio0_pins {
+			pcie0_pins: pcie0_pinmux {
 				mux {
-					pins = "gpio0", "gpio1";
-					function = "gpio";
-					drive-strength = <8>;
+					pins = "gpio3";
+					function = "pcie1_rst";
+					drive-strength = <12>;
```

Comment:
I check D7800 GPL and You have right. drive-strength is 2.
I remove this section.